.gitignore

```diff
@@ -85,7 +85,7 @@ out
 
 # Nuxt.js build / generate output
 .nuxt
-dist
+#dist
```

Comment:
Did you do this to work in BE with your types changes?

Reply:
Yes, I'll revert it when BE PRs are ready to be merged, only changes in `api.ts` will remain

Reply:
I think you can use `yarn link` so you don't have to check in the dist folder. Other than that, nice!